lib/portlayer/util/util.go

```diff
@@ -53,3 +57,15 @@ func ServiceURL(serviceName string) *url.URL {
 
 	return s
 }
+
+// Update the VM display name on vSphere UI
+func DisplayName(id, name string) string {
+	shortID := id[:shortIDLen]
+	nameMaxLen := maxVMNameLength - len(shortID)
+	prettyName := name
```

Comment:
do we need to make it smarter? like if pretty name is shorter than nameMaxLen, can we make the id longer than a shortID